README.md

```diff
@@ -1,2 +1,47 @@
 # openapi-python-client-action
-GitHub Action to generate a python client package from an openapi spec
+The official github action for [openapi-python-client](https://github.com/triaxtec/openapi-python-client) - generates a modern python client package from an openapi document
+
+## Inputs
+
+### `openapi-python-client-version`
+
+The version of the openapi-python-client package to use to generate the python client. If unspecified the latest released version will be used.
+
+### `openapi-file`
+
+The path (with respect to the current directory/the workspace) to the openapi document (both json and yaml are supported). Defaults to just "openapi.json" i.e. a file in the current directory called openapi.json.
```

Comment:
```suggestion
The path (with respect to the current directory/the workspace) to the OpenAPI document (both JSON and YAML are supported). Defaults to just "openapi.json" i.e. a file in the current directory called openapi.json.
```

Reply:
We may want to also allow a URL instead? IDK how common that would be to do in CI, so maybe not worth yet, but once we support using server config that might be more common?